great_expectations/cli/datasource.py

```diff
@@ -465,94 +636,72 @@ def create_sample_expectation_suite(
     msg_some_data_assets_not_found = """Some of the data assets you specified were not found: {0:s}    
     """
 
-    msg_prompt_enter_data_asset_name = "\nWhich data would you like to use? (Choose one)\n"
-
     msg_prompt_what_will_profiler_do = """
 Great Expectations will choose a couple of columns and generate expectations about them
 to demonstrate some examples of assertions you can make about your data. 
     
 Press any key to continue...
     """
 
+    msg_prompt_expectation_suite_name = """
+Name the new expectation sute"""
+
     msg_data_doc_intro = """
 <cyan>========== Data Docs ==========</cyan>"""
 
     if show_intro_message:
         cli_message(msg_intro)
 
-    batch_kwargs = None
-    data_asset_name = None
-    data_assets = []
-
-    #TODO: ["default"] is a hack. since we are running in init, it might be ok to assume that only default exists
-    available_data_assets = context.get_available_data_asset_names(datasource_names=[data_source_name])[data_source_name]["default"]
-    # TODO tell the user how many data assets are found and provide a sane picker
-    # print("Found {} datas".format(len(available_data_assets["names"])))
-    available_data_asset_names = ["{} ({})".format(name[0], name[1]) for name in available_data_assets["names"]]
+    data_source = select_datasource(context, data_source_name=data_source_name)
+    if data_source is None:
+        raise ge_exceptions.DataContextError("No datasources found in the context")
 
-    profiler = SampleExpectationsDatasetProfiler
+    data_source_name = data_source.name
 
-    if len(available_data_asset_names) == 0:
-        if "PandasDatasource" == type(context.get_datasource(data_source_name)).__name__:
-            data_asset_name, batch_kwargs = _load_file_as_data_asset_from_pandas_datasource(context, data_source_name)
-            data_assets = [data_asset_name]
-    else:
-        choices = "\n".join(["    {}. {}".format(i, name) for i, name in enumerate(available_data_asset_names[:5], 1)])
-        prompt = msg_prompt_enter_data_asset_name + choices + "\n"
-
-        selections = click.prompt(prompt, default=None, show_default=False)
-
-        data_asset_indices = set()
-        if selections:
-            # sanitize the inputs down to integers
-            selections = [item.strip() for item in selections.split(",")]
-            for item in selections:
-                try:
-                    data_asset_indices.add(int(item) - 1)
-                except ValueError:
-                    pass
-
-        data_assets = []
-        for i in data_asset_indices:
-            try:
-                data_assets.append(available_data_assets["names"][i][0])
-            except IndexError:
-                pass
+    if generator_name is None or data_asset_name is None or batch_kwargs is None:
+        data_source_name, generator_name, data_asset_name, batch_kwargs = get_batch_kwargs(context,
```

Comment:
It seems odd that we're requiring a user to get all the way to having batch_kwargs to 'create_expectation_suite'...but I think it also makes sense. Maybe we mean `create_expectation_suite_from_data_batch_definition` or something?

Reply:
Noting: we talked about this and want this to be a sort of _proto_ version of build suite from template/profiler/etc.

Reply:
All of these arguments are optional.